src/ReadableEnumInterface.php

```diff
@@ -32,6 +32,17 @@ public static function readables(): array;
      */
     public static function readableFor($value): string;
 
+    /**
+     * Gets the human representations for given values.
+     *
+     * @param array $values
+     *
+     * @throws InvalidValueException When $value is not acceptable for this enumeration type
+     *
+     * @return array
+     */
+    public static function readablesFor(array $values): array;
```

Comment:
Adding it to the interface if a BC break.
But Symfony's debug class loader allows to hint unimplemented `@methods` since https://github.com/symfony/symfony/pull/28902. So it could be a solution.